### DISCUSSION TITLE
CefSharp can't open https://scoopfit.com

### DISCUSSION THREAD

**Discussion by wu-zhi-qiang** [cefsharp/CefSharp#4317] [General]
CefSharp can't open https://scoopfit.com.

Clone and run CefSharp.WinForms.Example, visit 'www.sgcc.com.cn' show a blank page.

User cefclient and cefsample(C++), it's ok.

**Comment by amaitland** [comment]
See https://github.com/cefsharp/CefSharp/discussions/4304#discussioncomment-4185561

**Reply by wu-zhi-qiang** [reply]
thank you very much!

**Reply by wu-zhi-qiang** [reply]
solved, thank you .
